create_pr_comments.py

```diff
@@ -0,0 +1,87 @@
+import requests
+import json
+import os
+import logging
+from dotenv import load_dotenv
+from run_gemini import get_review
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(encoding="utf-8", level=logging.DEBUG)
+
+load_dotenv(override=True)
+
+
+def create_pull_request_comment(
+    repository, pull_number, github_api_token, body, commit_id, path, line, side
+):
+
+    pr_url = f"https://api.github.com/repos/{repository}/pulls/{pull_number}/comments"
+    headers = {
+        "Accept": "application/json",
+        "Authorization": f"Bearer {github_api_token}",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+    data = {
+        "body": body,
+        "commit_id": commit_id,
+        "path": path,
+        "line": int(line),
+        "side": side,
+    }
+
+    logger.debug(data)
+
+    response = requests.post(pr_url, headers=headers, json=data)
+    try:
+        response.raise_for_status()
+    except:
+        logger.exception(response)
+
+    return response.json()
```

Comment:
The function `create_pull_request_comment()` should handle the case when the API call fails with a non-200 response. This can be done by checking the response status code and raising an exception or logging the error appropriately.